build/Nuget/Microsoft.VisualStudio.ProjectSystem.VSInsertionNuget.proj

```diff
@@ -1,16 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\Targets\Settings.targets" />
-  
+  <ImportGroup>
+    <Import Project="..\Targets\Settings.targets" />
+    <Import Project="..\Targets\VSL.Versions.targets" />
+  </ImportGroup>
+
   <PropertyGroup>
-    <NuGetVersion>1.0.0.0</NuGetVersion>
-    <NuGetVersionType>InsertionPurposeOnly</NuGetVersionType>
-    <NuGetOutDir>$(OutDir)NuGet\$(NuGetVersionType)\</NuGetOutDir>
+    <NuGetOutDir>$(OutDir)DevDivPackages\Roslyn\</NuGetOutDir>
```

Comment:
Can we use a different term than "DevDivPackages"? This is a open source project, we shouldn't be using internal terms. I'd suggest just "Packages".